browser/resources/settings/brave_leo_assistant_page/model_config_ui.html

```diff
@@ -174,6 +174,36 @@
         error-message="[[invalidUrlErrorMessage]]"
       ></cr-input>
     </div>
+
+    <div class="input-container">
+      <div class="custom-label">
+        <div class="label-text">
+          <span>$i18n{braveLeoAssistantModelSystemPromptTitle}</span>
+          <leo-tooltip mode="default">
+            <div
+              slot="content"
+              class="tooltip-content"
+            >
+              $i18n{braveLeoAssistantModelSystemPromptDesc}
+            </div>
+            <leo-button
+              size="small"
+              kind="plain-faint"
+              fab
+            >
+              <cr-icon class="small"  icon="info-outline"></cr-icon>
```

Comment:
could you try using a `leo-icon` here instead - I've noticed some of the sizing is a bit funky with a `cr-icon` focus ring:


```suggestion
              <leo-icon name="info-outline"></cr-icon>
```

(size will hopefully be implied by the button size)

Reply:
Looks like the leo-icon version is positioned a bit higher than the cr-icon elements. 

leo-icon:

![image](https://github.com/user-attachments/assets/ea841c59-a3a5-4e72-b0e8-9ef0aea13565)

cr-icon:

![image](https://github.com/user-attachments/assets/baa4e4f5-8ab5-4eb4-9547-d31d5615a3c6)

Reply:
Ah feel free for either then :)